From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Release Engineering <releng@openela.org>
Date: Fri, 10 Jul 2026 08:47:16 +0000
Subject: [PATCH] Update OpenELA debranding

---
 SPECS/osbuild-composer.spec | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/SPECS/osbuild-composer.spec b/SPECS/osbuild-composer.spec
index 3321cd9..a9dbdcf 100644
--- a/SPECS/osbuild-composer.spec
+++ b/SPECS/osbuild-composer.spec
@@ -188,6 +188,14 @@ done
 %endif
 %endif
 
+%if 0%{?openela}
+%if 0%{?openela} >= 9
+install -m 0644 -vp repositories/openela-*                           %{buildroot}%{_datadir}/osbuild-composer/repositories/
+%else
+install -m 0644 -vp repositories/openela-8*                          %{buildroot}%{_datadir}/osbuild-composer/repositories/
+%endif
+%endif
+
 # Fedora can build for all included fedora releases
 %if 0%{?fedora}
 install -m 0644 -vp vendor/github.com/osbuild/images/data/repositories/fedora-*                          %{buildroot}%{_datadir}/osbuild-composer/repositories/
--
2.47.3
