From 8b15cc4ffa92b266a2dc9ec79395bb7b90d8a1df Mon Sep 17 00:00:00 2001
From: Mustafa Gezen <mustafa@ctrliq.com>
Date: Wed, 15 Nov 2023 19:08:17 +0100
Subject: [PATCH] OpenELA patch

---
 SPECS/osbuild-composer.spec | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/SPECS/osbuild-composer.spec b/SPECS/osbuild-composer.spec
index 0b9bb99..13afa8c 100644
--- a/SPECS/osbuild-composer.spec
+++ b/SPECS/osbuild-composer.spec
@@ -154,6 +154,14 @@ install -m 0644 -vp repositories/rhel-%{rhel}*                     %{buildroot}%
 %endif
 %endif
 
+%if 0%{?openela}
+%if 0%{?openela} >= 9
+install -m 0644 -vp repositories/openela-*                         %{buildroot}%{_datadir}/osbuild-composer/repositories/
+%else
+install -m 0644 -vp repositories/openela-8*                        %{buildroot}%{_datadir}/osbuild-composer/repositories/
+%endif
+%endif
+
 # Fedora can build for all included fedora releases
 %if 0%{?fedora}
 install -m 0644 -vp repositories/fedora-*                          %{buildroot}%{_datadir}/osbuild-composer/repositories/
-- 
2.39.3 (Apple Git-145)


