From 253818b8c7c778710c51225f8d492438813b9c94 Mon Sep 17 00:00:00 2001
From: Alex Burmashev <alexander.burmashev@oracle.com>
Date: Tue, 11 Nov 2025 12:12:36 +0000
Subject: [PATCH] Update OpenELA debranding

---
 SPECS/osbuild-composer.spec | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/SPECS/osbuild-composer.spec b/SPECS/osbuild-composer.spec
index 98b8267..2cf7f6d 100644
--- a/SPECS/osbuild-composer.spec
+++ b/SPECS/osbuild-composer.spec
@@ -186,6 +186,14 @@ done
 %endif
 %endif

+%if 0%{?openela}
+%if 0%{?openela} >= 9
+install -m 0644 -vp repositories/openela-*                           %{buildroot}%{_datadir}/osbuild-composer/repositories/
+%else
+install -m 0644 -vp repositories/openela-8*                          %{buildroot}%{_datadir}/osbuild-composer/repositories/
+%endif
+%endif
+
 # Fedora can build for all included fedora releases
 %if 0%{?fedora}
 install -m 0644 -vp vendor/github.com/osbuild/images/data/repositories/fedora-*                          %{buildroot}%{_datadir}/osbuild-composer/repositories/
--
2.47.3
