From a4dbdceaf8634a5c4548d9bd24f6e9aaa4e8f14a Mon Sep 17 00:00:00 2001
From: Mustafa Gezen <mustafa@ctrliq.com>
Date: Tue, 5 Mar 2024 20:56:42 +0100
Subject: [PATCH] OpenELA patch

---
 SPECS/osbuild-composer.spec | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/SPECS/osbuild-composer.spec b/SPECS/osbuild-composer.spec
index db838c2..223dfa4 100644
--- a/SPECS/osbuild-composer.spec
+++ b/SPECS/osbuild-composer.spec
@@ -153,6 +153,14 @@ install -m 0644 -vp repositories/rhel-%{rhel}*                     %{buildroot}%
 %endif
 %endif

+%if 0%{?openela}
+%if 0%{?openela} >= 9
+install -m 0644 -vp repositories/openela-*                           %{buildroot}%{_datadir}/osbuild-composer/repositories/
+%else
+install -m 0644 -vp repositories/openela-8*                          %{buildroot}%{_datadir}/osbuild-composer/repositories/
+%endif
+%endif
+
 # Fedora can build for all included fedora releases
 %if 0%{?fedora}
 install -m 0644 -vp repositories/fedora-*                          %{buildroot}%{_datadir}/osbuild-composer/repositories/
--
2.39.3 (Apple Git-145)
